packages/beacon-node/src/chain/validatorMonitor.ts

```diff
@@ -34,6 +34,7 @@ const MAX_CACHED_EPOCHS = 4;
 
 const MAX_CACHED_DISTINCT_TARGETS = 4;
 
+// TODO GLOAS: update to use BPS and re-evaluate values
```

Comment:
we might already wanna do this as part of the PR, there really shouldn't be any remaining usage of `INTERVALS_PER_SLOT`

Reply:
will be addressed by https://github.com/ChainSafe/lodestar/pull/8471